spec/workers/save_to_cloud_storage_worker_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+RSpec.describe SaveToCloudStorageWorker, type: :worker do
+  let(:worker) { described_class.new }
+
+  describe "#perform" do
+    let(:asset) { FactoryGirl.create(:clean_asset) }
+
+    context 'when S3 bucket is configured' do
+      let(:cloud_storage) { double(:cloud_storage) }
+
+      before do
+        allow(Services).to receive(:cloud_storage).and_return(cloud_storage)
+      end
+
+      it 'saves the asset to cloud storage' do
+        expect(cloud_storage).to receive(:save).with(asset)
+
+        worker.perform(asset)
+      end
+
+      context 'when an exception is raised' do
+        let(:exception_class) { Class.new(StandardError) }
+        let(:exception) { exception_class.new }
+
+        before do
+          allow(cloud_storage).to receive(:save).and_raise(exception)
+        end
+
+        it 'reports the exception to Errbit via Airbrake' do
+          expect(Airbrake).to receive(:notify_or_ignore)
+          .with(exception, params: { id: asset.id, filename: asset.filename })
```

Comment:
Very minor: I think I'd expect this line to be indented two more spaces. Does Rubocop not suggest that?

Reply:
No - it seems to pass a full check locally and on CI.